database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/MyPostsView.swift

```diff
@@ -31,6 +31,9 @@ struct MyPostsView: View {
       .onAppear {
         user.getPosts(tabOpened: tabOpened)
       }
+      .onDisappear {
+        user.removeRefHandles()
```

Comment:
Just a small correction. I don't think the view needs to know about ref handles as it is an implementation detail. The view should just forward the events important for the View Model. For this case it basically means renaming the method `UserViewModel.removeRefHandles()` to `UserViewModel.onViewDisappear()`.  The same applicable for `onAppear` here and for other views below.